src/rename_samples.py

```diff
@@ -0,0 +1,22 @@
+import click
+
+from wrappers.cobs import Cobs
+
+
+@click.command()
+@click.argument('infile', type=click.File('r'))
+def rename_samples(infile):
+    mapping = {}
+
+    lines = infile.readlines()
```

Comment:
```suggestion
```

Reply:
Don't try to read the whole file for line based file even though the file is expected to be small. The file object is a lazy generator of lines.